### PR TITLE
Don't run live stream until after we finish write the seq

### PR DESCRIPTION
**Pull request by arj03**
One more of these *facepalm* bugs. With this, the index time of private (that I'm working on) goes down from 10 seconds to 2 seconds :-) And social from 12 to 7.

### CODE REVIEW

**Comment by staltz** [comment]
Awesome gains! Can totally merge. Have one inline comment though

File: indexes/plugin.js [hunk 1]
```diff
@@ -61,6 +61,15 @@ module.exports = function (
       }
     }
 
+    function liveStream()
+    {
```
Comment (staltz) [inline indexes/plugin.js:65]: I'm curious why didn't the code formatter git hook apply here? It should run automatically when you commit

Comment (arj03) [reply indexes/plugin.js:65]: Yeah that it strange, I just ran it manually and it made some minor changes. Will commit and merge.

Comment (arj03) [reply indexes/plugin.js:65]: oh, maybe its because it is only running on staged files? I just git commit -a

**Pull request merged**